tests/Examples/Fingerprint/FingerprintTaskWithAFingerprintAndForceTest.php.output_not_runnable.txt

```diff
@@ -1,2 +1,5 @@
+hh:ii:ss WARNING   [castor] Remote imports are disabled, skipping import of "pyrech/castor-example".
+hh:ii:ss WARNING   [castor] Remote imports are disabled, skipping import of "pyrech/castor-example-package-not-published".
+hh:ii:ss WARNING   [castor] Remote imports are disabled, skipping import of "pyrech/foobar".
```

Comment:
Instead of this, could you update tests/Helper/OutputCleaner.php to **remove** theses 3 lines?